src/index.ts

```diff
@@ -127,9 +127,9 @@ export class SunsynkPowerFlowCard extends LitElement {
         const state_day_aux_energy = this.getEntity('day_aux_energy');
 
         //Inverter
-        const state_inverter_voltage = this.getEntity('inverter_voltage_154', {state: '0'});
-        const state_load_frequency = this.getEntity('load_frequency_192', {state: '0'});
-        const state_inverter_current = this.getEntity('inverter_current_164', {state: '0'});
+        const state_inverter_voltage = this.getEntity('inverter_voltage_154');
```

Comment:
The getEntity function defaults to a state of 0, is the the reason for tremoving the defult value {state: '0'}. If yes can we remove it everywhere?

Reply:
Yes you are correct, since the method already defaults tk that, it was removed.